.gitignore

```diff
@@ -7,6 +7,8 @@
 
 # testing
 /coverage
+/cypress/screenshots
+/cypress/videos
```

Comment:
what are these for?

Reply:
we need to ignore this path as it captures the screenshot and video of the test which runs in headless mode, and of course we don't want to include those in Github :)